### ISSUE TITLE
Blog

### ISSUE THREAD

**Issue by katiehayman** [SmackBang/tmbrmll#4]
Is there an option for one of the images to be a sliding gallery?

**Comment by chausme** [comment]
No unfortunately there is no such option, could you please tell do you mean Blog page or Single post page?

**Comment by katiehayman** [comment]
within the single blog post.

We did it with the Vista (top image is an option to be a gallery) http://followthevista.com/london-calling-stylish-guru-alchemists-ondine-daisy/

Does that make sense?

**Comment by chausme** [comment]
Of course, do we need this function for featured image only or for inline images as well?

**Comment by katiehayman** [comment]
is there a preferred way for you ?

**Comment by chausme** [comment]
We can go both ways but implementing inline gallery will take more time

**Comment by katiehayman** [comment]
have these changes been made?

**Comment by chausme** [comment]
Can you please tell which way we should go:

> is there a preferred way for you ?

> We can go both ways but implementing inline gallery will take more time

**Comment by katiehayman** [comment]
haha sorry! go the easier way :)

**Comment by chausme** [comment]
Thanks! I will work on the changes tomorrow, please add any feedback within today :)

**Comment by katiehayman** [comment]
sure! will have another look but think ive added a bit for now :)
